site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -29,36 +29,47 @@ const getRandomGradeableName = () => {
     }
     return randomString;
 };
+const makeNonTeamGradeable = (gradeableName) => {
+    cy.login('instructor');
+    cy.visit(['sample', 'gradeable']);
+    // Enter gradeable info
+    cy.get('[data-testid=create-gradeable-title]').type(gradeableName);
+    cy.get('[data-testid=create-gradeable-id]').type(gradeableName);
+    cy.get('[data-testid=radio-student-upload]').check({ force: true });
```

Comment:
Is there a way to do this without force clicking?

Reply:
it says DOM element is not visible despite it being clearly visible

Reply:
When testing locally it wouldn't check the button without force. Could you add a comment above it saying 'force is currently necessary to ensure the radio button is checked' or something similar, so when people come along later and try and remove force clicks, they know why this one is here?

Reply:
There are many situations in this file wherewe use .check() and then .click(), which works. Would you want all the test to refactor using a single check and then force?

Reply:
I think leaving them for now is fine, Ideally we wouldn't force click (most things), but I think leaving this one is fine, and maybe later on when someone tries to remove all force clicks, they'll change it to check then click.